src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/AssessmentEntity.kt

```diff
@@ -362,6 +365,7 @@ interface DomainAssessmentSummary {
   val crn: String
   val status: DomainAssessmentSummaryStatus?
   val dueAt: Timestamp?
+  val probationDeliveryUnitName: String?
```

Comment:
Presumably this is ok to add here?  `DomainAssessmentSummary` is used across CAS1/3.